viewutil/viewutil.go

```diff
@@ -108,7 +109,7 @@ func (bd *BaseData) withBaseValues(meta Meta, headerLinks []HeaderLink, commonSc
 
 // Base is a temporary wrapper around BaseEn and BaseRu, meant to facilitate the migration from qtpl.
 // TODO: get rid of this
-func Base(meta Meta, title, body string, headElements ...string) string {
+func Base(meta Meta, title, body string, extraData []string, headElements ...string) string {
```

Comment:
OK. It is cumbersome to pass the empty `[]string{}`, but whatever, `Base` is to be deprecated.

Reply:
Yeah I'm also not a fan of it, but I couldn't think of anything better at the time.